spark3/src/main/java/org/apache/iceberg/spark/actions/BaseRewriteDataFilesSpark3Action.java

```diff
@@ -35,6 +36,11 @@ protected BinPackStrategy binPackStrategy() {
     return new Spark3BinPackStrategy(table(), spark());
   }
 
+  @Override
+  protected SortStrategy sortStrategy() {
```

Comment:
This is a little strange to me. It feels more like a `createSortStrategy()`, but I think it's fine since it isn't exposed.

Reply:
Yeah this is just copying what we did for binPack ^ but we can change them both, if needed